parachain/primitives/router/src/outbound/mod.rs

```diff
@@ -132,7 +132,7 @@ where
 		})?;
 
 		// convert fee to MultiAsset
-		let fee = MultiAsset::from((MultiLocation::parent(), fee)).into();
+		let fee = MultiAsset::from((MultiLocation::parent(), fee.total())).into();
```

Comment:
```suggestion
		let fee = MultiAsset::from((MultiLocation::parent(), fees.total())).into();
```

Reply:
[Rename to fees](https://github.com/Snowfork/snowbridge/pull/975/commits/356d17c9b8c57a14e6d411185d1473c942b4579d)